docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,178 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(masternode_announce_message)
+  
+**Result**
+
+  A message with the result of the broadcasting, it could be a success message or an error one including the reason of failure.
+
+masternode.subscribe
+--------------
+
+Returns the status of masternode.
+
+**Signature**
+
+  .. function:: masternode.subscribe(masternode_collateral)
+
+**Result**
+
+  As this is a subcription, the client will receive a notification when the masternode status changes.
+
+masternode.list
+--------------
+
+Returns the list of masternodes.
+
+**Signature**
+
+  .. function:: masternode.list(payees)
+  
+  * *payees*
+
+    A single string or an array of masternode payee addresses.
+
+**Result**
+
+  An array with the masternodes information.
+  
+  * If the payee is empty, all the masternodes in the network will be retrieved.
```

Comment:
I assume this means an empty array?  Or does it mean missing?  Or does it mean either?  Please clarify.

Please refer to the parameter as "payees" as that is what you named it, not "payee".